### ISSUE TITLE
[Google sheets add-on] Sidebar "Set API Key" and other things are NOT doing anything on click

### ISSUE THREAD

**Issue by giorgosera** [ulangi/ulangi#73]
Hey there,

I installed the google sheets add-on and gave all the permissions requested. Still the Set API Key button does absolutely nothing when clicked. Also, the other buttons/links (except Push and Pull) are not clickable.

I'm using Chrome on MacOS.

**Comment by giorgosera** [comment]
**Update**

The add on works fine in a Google Apps account (my company's account) but doesn't work at all for my personal gmail account.

Opening the console I see this:

![image](https://user-images.githubusercontent.com/752207/80714659-e3b6fa00-8afd-11ea-9e04-ec64c2eb6784.png)

**Comment by minhloi** [comment]
@giorgosera I haven't investigated this issue yet but this looks like the issue is from Google not the add-on. Google has been migrating add-ons from Chrome Web Store to G Suite Marketplace and it caused many bugs. Some users said that it works fine on Firefox but not on Chrome.

**Comment by giorgosera** [comment]
@minhloi yes you are right! Didn't test it on Firefox but it works on Safari. 

Thanks a lot Google :)
